### PR TITLE
Add RSS feed of new releases to dist.ipfs.io

### PR DESCRIPTION
**Pull request by neatonk**
See #241

### CODE REVIEW

**Comment by jessicaschilling** [comment]
Hey @neatonk -- sorry for the delay on replying to this, I think I accidentally cleared a notification. Is it ready for review?

**Comment by neatonk** [comment]
Yes, ready for review.

**Comment by jessicaschilling** [comment]
Gateway link for review is here (built from my local of this PR):
https://gateway.ipfs.io/ipfs/QmQmvMzK1JFQyFS6YiKhTggr2wejLhLURH1AoSfmeRk1Jy

@Stebalien -- are you able to make a review? I've gone over it from the end-user perspective and it behaves as expected; just have one visual comment in my own review.

Thanks!

File: site/layouts/_default/single.html [hunk 1]
```diff
@@ -12,8 +12,15 @@
   <body data-spy="scroll" data-target="#d-navbar" data-offset="40">
     {{ partial "header.html" }}
     <div class="container">
-      <div id="about" class="row d-about col-md-9 offset-md-3">
-        {{ partial "about.md" | markdownify }}
+      <div class="row">
+        <div id="about" class="d-about col-md-9 offset-md-3">
+          <div class="d-rss pl-3 float-right">
+            <a href="index.xml">
+              <img style="height: 1em" src="./img/icon-rss.svg">
```
Comment (Stebalien) [inline site/layouts/_default/single.html:19]: Any reason not to specify the height in the CSS?

File: scripts/dist.js [hunk 2]
```diff
@@ -31,6 +31,9 @@ function getVersion (type, done) {
 function writeData (type, version, done) {
   const dataPath = join(RELEASE_PATH, type, version, 'dist.json')
   const data = JSON.parse(fs.readFileSync(dataPath).toString())
+  if (!data.dateUTC && data.date) {
+    data.dateUTC = new Date(data.date).toUTCString()
```
Comment (Stebalien) [inline scripts/dist.js:35]: I'd prefer to do this in the template rather than sneak it in here, but it's not a huge issue either way.

**Comment by neatonk** [comment]
> This looks great overall! From examining https://gateway.ipfs.io/ipfs/QmQmvMzK1JFQyFS6YiKhTggr2wejLhLURH1AoSfmeRk1Jy (gateway-pinned instance from my local copy of this PR)

Thanks so much for building and pinning that.

> one small visual request: Possible to move the RSS icon/text onto its own line at narrower viewports so as to avoid awkward white space like the following?

Done. See https://github.com/ipfs/distributions/pull/312/commits/78227a4ea8e0263238d5463988302b4bce4baf5e.

Also here are some screenshots for review:

Screen Shot 2020-08-18 at 21 06 28
Screen Shot 2020-08-18 at 21 06 36
Screen Shot 2020-08-18 at 21 06 51
Screen Shot 2020-08-18 at 21 07 03
Screen Shot 2020-08-18 at 21 07 41
Screen Shot 2020-08-18 at 21 08 00

File: site/layouts/_default/single.html (hunk 1 above)
Comment (neatonk) [reply site/layouts/_default/single.html:19]: Yes, good question. Putting the height on the image tag ensures that the image will have a reasonable height, even if the CSS fails to load.

Comment (Stebalien) [reply site/layouts/_default/single.html:19]: Ah, especially while loading. Got it.

File: scripts/dist.js (hunk 2 above)
Comment (neatonk) [reply scripts/dist.js:35]: Me too. I initially tried to use the [`dateFormat`](https://gohugo.io/functions/dateformat/) function provided by Hugo, but it cannot parse the date format used in the `data.json` files. 

Another option would be to change the date format used in the `data.json` files so they can be formatted as needed using the `dateFormat` function.

Comment (Stebalien) [reply scripts/dist.js:35]: Ah... and we can't define a custom function. Eh, nevermind.

**Pull request merged**